examples/plot_tensorflow.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+=================================
+An example for tensorflow
+=================================
+"""
+# %%
+
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import accuracy_score, precision_score, recall_score
+import scikeras
+import warnings
+from tensorflow import get_logger
+import numpy as np
+from scikeras.wrappers import KerasClassifier
+from tensorflow import keras
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import sklearn.metrics as skm
+%matplotlib inline
+import fairlearn
+from fairlearn.metrics import selection_rate
+
+from fairlearn.metrics import (
+    MetricFrame,
+    false_positive_rate,
+    true_positive_rate,
+    selection_rate
+)
+
+get_logger().setLevel('ERROR')
+warnings.filterwarnings("ignore", message="Setting the random state for TF")
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X_raw = data.data
+Y = (data.target == '>50K') * 1
+sex = data.data["sex"]
+
+X = X_raw.drop(labels=['sex'], axis=1)
+X = pd.get_dummies(X)
+
+def get_clf(meta, hidden_layer_sizes, dropout):

```

Comment:
It would be good to have some explanation as to what's going on here, if you want to keep this example separate.

Reply:
@riedgar-ms Sure, I will add some comments that would explain the purpose of the code line, in `plot_tensorflow.py`.

I did not do this yet because I am confused if this example would be used for the original directory or not because it has got some libraries that are not aligning with the dependencies which are already there as mentioned in `requirements-dev.txt`. And as per the earlier discussions, I see that the admins do not wish to add dependencies upon the newly used libraries.